### DISCUSSION TITLE
Using proton GE

### DISCUSSION THREAD

**Discussion by Voidwalker60** [RareDevs/Rare#79] [Q&A]
How would I use proton GE? I don't see it in the dropdown menu.

**Comment by Dummerle** [comment]
Rare searches in a few directories from steam. An option for custom proton versions is not implemented yet.

You can edit the wrapper manually, and you have to add no-wine option in ~/.config/legendary/config.ini

**Comment by ChemicalXandco** [comment]
You should set `~/.config/legendary/config.ini` to:

```
[default]
wrapper = "/path/to/proton ge/proton" run
no_wine = true

[default.env]
STEAM_COMPAT_DATA_PATH = /home/[your username]/.proton
```

as described at https://github.com/derrod/legendary/wiki/Using-Proton
